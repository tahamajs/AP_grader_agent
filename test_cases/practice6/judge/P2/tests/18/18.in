POST signup ? username "amir" password "rfz82"
POST increase_budget ? amount 1000
POST reserve ? restaurant_name "yas" table_id "3" start_time "13" end_time "15" foods "joojeh,kabab koobideh"
POST reserve ? restaurant_name "lidoma" table_id "1" start_time "15" end_time "17" foods "joojeh"
GET reserves ?
GET show_budget ?