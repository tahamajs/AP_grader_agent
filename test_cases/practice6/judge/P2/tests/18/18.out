OK
OK
Reserve ID: 1
Table 3 for 13 to 15 in yas
Original Price: 510
Order Amount Discount: 40
Total ItemSpecific Discount: 108
First Order Discount: 40
Total Discount: 188
Total Price: 322
Reserve ID: 1
Table 1 for 15 to 17 in lidoma
Original Price: 320
Order Amount Discount: 0
Total ItemSpecific Discount: 0
First Order Discount: 40
Total Discount: 40
Total Price: 280
1: yas 3 13-15 joojeh(1) kabab koobideh(1) 510 322
1: lidoma 1 15-17 joojeh(1) 320 280
398